### PR TITLE
Uncraftable as synonym of Non-Craftable & Discord webhook integration

### PR DESCRIPTION
**Pull request by ZeusJunior**

### CODE REVIEW

File: app/handler/trades.js [hunk 1]
```diff
@@ -841,6 +842,9 @@ exports.offerChanged = function (offer, oldState) {
 
         if (offer.state === TradeOfferManager.ETradeOfferState.Accepted) {
             admin.message('Trade #' + offer.id + ' with ' + offer.partner.getSteamID64() + ' is accepted. Summary:\n' + offer.summarize());
+            if (process.env.DISCORD_WEBHOOK_URL) {
```
Comment (Hhanuska) [inline app/handler/trades.js]: You used upper case 'L' here in `DISCORD_WEBHOOK_URL` but lower case everywhere else.

File: app/discord.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const Discord = require('discord.js');
+const hookid = process.env.DISCORD_WEBHOOK_URl.split('/')[5];
+const hooktoken = process.env.DISCORD_WEBHOOK_URl.split('/')[6];
```
Comment (Hhanuska) [inline app/discord.js]: These will return an error if users update the bot, but don't add `DISCORD_WEBHOOK_URl=""` to the .env file. I'm not sure if we should worry about that though. @Nicklason ?

File: app/handler/trades.js (hunk 1 above)
Comment (ZeusJunior) [reply app/handler/trades.js]: Lol idk how that happened, should be uppercase everywhere

File: app/discord.js (hunk 2 above)
Comment (ZeusJunior) [reply app/discord.js]: I thought about adding a shorthand conditional statement here. Decided not to, if they don’t want to use the discord msgs they should just leave it at “”

Comment (Nicklason) [reply app/discord.js]: It should not make the bot crash if you don’t include it, so please fix that

**Comment by ZeusJunior** [comment]
Might add when the socket server gets set up in x years instead

**Pull request closed without merging**